### PR TITLE
Updated The README.md

### LINKED ISSUE #2: [DOCS]: Update README.md
### Context
The `README.md` file is out of date, ugly, and probably missing import information xD. Let's fix that! Check out some of my other repos for my typical style of `README` or get creative!

### Acceptance Criteria
- Fork this repository and make a new branch
- Update the `README.md` (content wise and aesthetic wise!)
- Create a PR

<br>

Happy coding! ❤️ 
-Mandy

### PR DESCRIPTION
**Pull request by danieltayal07**
> Fixes #2 

added a few additional informations 
added emojis to make it aesthetically pleasing 
clearer steps
formatting

<br>

#### Screenshots and/or Screen Recordings

Screenshot 2024-10-19 at 3 14 00 AM
Screenshot 2024-10-19 at 3 14 08 AM
Screenshot 2024-10-19 at 3 14 16 AM
Screenshot 2024-10-19 at 3 14 23 AM
Screenshot 2024-10-19 at 3 14 29 AM

<br>

#### Checklist before requesting a review
- [✅ ] I have read through `CONTRIBUTING.MD`
- [ ✅] I have checked that no other PRs are out for the same issue
- [✅ ] I have performed a self-review of my code
- [✅ ] I have commented my code
- [✅ ] I have tested my code
- [✅ ] I have ensured all dependent changes have already been merged
- [ ✅] I know this code generates no new errors or warning

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,97 +1,181 @@
...
+#### ✅ Step 2: Make it Executable
 
-```
+Ensure the file is executable by running the following command in the directory where you've downloaded it:
+
+```bash
 chmod +x mdtoc
 ```
 
 <br>
```
Comment (Mandy-cyber) [inline README.md:48]: Can you remove the `br`s across the file? I think they're adding a bit too much whitespace. Feel free to keep them though in places where you think the whitespace is necessary

File: README.md [hunk 2]
```diff
@@ -1,97 +1,181 @@
...
 
-##### Step 3 (optional)
-If you want to make the generator executable globally on Linux do.
 
+## 💡 Tip: You can check if the file is executable by running:
```
Comment (Mandy-cyber) [inline README.md:51]: Please make the tip section smaller 😄

File: README.md [hunk 3]
```diff
@@ -1,97 +1,181 @@
...
+
+And voilà! 🎉 You now have a neatly organized and clickable ToC!
+
+## 🔧 Additional Information
+
+- **Supported Markdown Headers**: The generator will include headers from # to ###### in the ToC.
+
+- **Customization**: Feel free to modify the code to adjust the ToC style (e.g., nested lists, numbering).
+
+- **Issues & Contributions**: If you encounter any issues or have ideas for improving the tool, please open a GitHub issue or submit a pull request. We welcome contributions! 🙌
```
Comment (Mandy-cyber) [inline README.md:172]: Can you link this to the `CONTRIBUTING.md` file?

File: README.md (hunk 1 above)
Comment (danieltayal07) [reply README.md:48]: sure will do right away!

Comment (danieltayal07) [reply README.md]: done!

File: README.md (hunk 2 above)
Comment (danieltayal07) [reply README.md]: done!

File: README.md (hunk 3 above)
Comment (danieltayal07) [reply README.md]: done linked both the points to CONTRIBUTING.md !

**Review by Mandy-cyber — APPROVED** [review]
Great work, thanks for your contribution!

**Comment by danieltayal07** [comment]
Thank youu! for the opportunity @Mandy-cyber  ❤️

**Pull request merged**